### PR TITLE
DOC-10605: Added maximum query formula

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/install/pages/sizing-general.adoc [hunk 1]
```diff
@@ -362,6 +362,9 @@ There are a few questions that will help size the cluster:
 Different queries have different resource requirements.
 A simple query might return results within milliseconds while a complex query may require several seconds.
 
+The number of queries that may be processed simultaneously may be approximated with the formula _CPU_cores * 4_.
```
Comment (kartika-nair) [inline modules/install/pages/sizing-general.adoc]: This should probably be changed to the ASCII underscore:
```suggestion
The number of queries that may be processed simultaneously may be approximated with the formula _CPU&#95;cores * 4_.
```

**Review by kartika-nair — CHANGES_REQUESTED** [review]

File: modules/install/pages/sizing-general.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/install/pages/sizing-general.adoc]: Sorry, missed that. Thanks.

Comment (tonyjhillman) [reply modules/install/pages/sizing-general.adoc]: Is this required for prior releases also?..I assume so.

Comment (kartika-nair) [reply modules/install/pages/sizing-general.adoc]: Yes, I believe this calculation applies to all versions that aren't End of Life at the moment (Couchbase Server 6.6 and above).

**Pull request merged**